util/Migrator/DbScripts/2023-01-23_00_AddAvatarToOrganizationUsers.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR ALTER VIEW [dbo].[OrganizationUserUserDetailsView]
```

Comment:
I recently modified a view, and I had an enterprise team member show me that view changes can require metadata adjustments: https://contributing.bitwarden.com/contributing/code-style/sql/#adjusting-metadata-1